Explorer/Assets/Scripts/Global/Dynamic/MainSceneLoader.cs

```diff
@@ -272,6 +277,27 @@ await bootstrap.InitializeFeatureFlagsAsync(bootstrapContainer.IdentityCache!.Id
             }
         }
 
+        private async UniTask<bool> CheckPlayerBlocked(StaticContainer sc, string playerID, SplashScreen splashScreen, IWebBrowser webBrowser, IDecentralandUrlsSource urlsSource, CancellationToken ct)
+        {
+            bool isBlocklisted = await ApplicationBlocklistGuard.CheckIfBlocklistedAsync(sc.WebRequestsContainer.WebRequestController, urlsSource, playerID, ct);
+
+            if (!isBlocklisted) return false;
+
+            splashScreen.Hide();
```

Comment:
I would extract this UI logic in separate method. Anyway it is in the (`isBlockListed == true`) branch.  So it would be something like that 

 ```
       if (await ApplicationBlocklistGuard.IsPlayerBlocklistedAsync())
                {
                     await ShowBlockedScreen();
                     return;
                }
```

Reply:
Ah if we do it like this then it makes sense, ok will do, ignore previous comment 😄